lerobot/common/policies/act/modeling_act.py

```diff
@@ -38,7 +38,11 @@
 from lerobot.common.policies.normalize import Normalize, Unnormalize
 
 
-class ACTPolicy(nn.Module, PyTorchModelHubMixin):
+class ACTPolicy(nn.Module, PyTorchModelHubMixin, 
+                library_name="lerobot",
+                repo_url="https://github.com/huggingface/lerobot",
+                tags=["act-policy", "robotics"],
```

Comment:
Since `"act-policy"` is added as a tag here, it would make sense to add the policy as a tag for `DiffusionPolicy`, `TDMPCPolicy` and `VQBeTPolicy` as well.